markdown/developers/adding_pipelines.md

```diff
@@ -74,7 +74,7 @@ git push
 
 ### Set up Travis and Docker Hub
 
-The nf-core pipelines use two additional services that link to GitHub: Travis and Dockerhub.
+The nf-core pipelines use two additional services that link to GitHub: Travis and Docker Hub.
```

Comment:
Hmm, Travis eh?? I guess we should probably update these docs!

Is that something you fancy doing whilst you're here? Strip out all mention of Travis and mention GitHub Actions instead (which I think just work? Or does it need activating?)

Reply:
I tried to rewrite the documentation to fit GitHub Actions and remove all mentions of Travis.